src/themes/color.rs

```diff
@@ -6,70 +6,80 @@ use std::fmt;
 use std::ops::Add;
 use std::str::FromStr;
 
-/// An RGB color (red, green, blue).
+/// An RGBA color (red, green, blue, alpha).
 #[derive(Copy, Clone, Debug, Default)]
-pub struct Rgb {
+pub struct Rgba {
     pub r: f64,
```

Comment:
Do we need RGB components to be floating point numbers?

Reply:
We don't have to use `f64`s for `Rgba`, but it does require more casting for `Rgba` <-> `Hsva`. I'll put add an additional commit for review.